payara-maven/README.md

```diff
@@ -0,0 +1,86 @@
+# Demonstration of using Payara Server with the Maven Cargo plugin
```

Comment:
I would change the title to mention "deploying to Payara Server", for example:
"Demonstration of deploying an application to Payara Server with the Maven Cargo plugin"